Python/Image_Sketch/README.md

```diff
@@ -0,0 +1,10 @@
+# Image to Sketch
+The original image is converted into black and white sketched image.
+
```

Comment:
add requirements to run this script. Add command how to use this script. Please look for contribution guidelines before sending PRs. You need to write proper readme documentation.

Reply:
I have updated readme.md with the appropriate changes. Can you please review it again? @tejan-singh